packages/app/package.json

```
@@ -138,7 +138,7 @@
   },
   "scripts": {
     "analyze": "cross-env ANALYZE=true yarn build",
-    "bootstrap": "exit 0",
+    "bootstrap": "yarn download && yarn workspace @corona-dashboard/cms lokalize:export",
```

```suggestion
    "bootstrap": "yarn download",
```

The `lokalize:export` script is already ran from the `@corona-dashboard/cms` package when you execute `yarn bootstrap` from the monorepo root.

---

We removed `yarn download` from the bootstrap command since we often work with development datasets that will be overwritten by the command, while it is a very useful command during development as well. We should update the documentation to make that clear though.

---

OK, so I take this issue can then be entirely resolved by updating the README? I've reverted the change that updated `package.json` and updated the README instead:

1. I removed the `Yarn Bootstrap` section, since it falsely claimed the download and lokalize scripts would be run
2. I updated the listed commands to not include `yarn bootstrap`, but to _do_ include all other relevant commands
3. Some other minor updates

This way, we make no changes to Yarn scripts and local develop environments would not be impacted while still allowing others to properly build and run locally by following the instructions in the README.

Does that work for you?